docs/source/remote-data-services.rst

```diff
@@ -0,0 +1,155 @@
+Remote Data Services
+====================
+
+As described in Section :doc:`Internal Data Ingestion <bytes>`, various
+user-facing functions such as ``dataframe.read_parquet`` and lower level
+byte-manipulating functions may point to data
+that lives not on the local storage of the workers, but on a remote system
+such as Amazon S3.
+
+In this section we describe how to use the various known back-end storage
+systems. Below we give some details for interested developers on how further
+storage back-ends may de provided for dask's use.
+
+
+Known Storage Implementations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When specifying a storage location, a URL should be provided using the geneal
+form ``protocol://path/to/data``.
+If no protocol is provided, the local file-system is assumed (same as
+``file://``). Two methods exist for passing parameters to the backend
+file-system driver: extending the URL to include username, password, server,
+port, etc.; and providing ``storage_options``, a dictionary of parameters
+to pass on.
```

Comment:
Relevant formatting

```
.. code-block:: python
<full line break>
   python code
```

Note that there are *three* spaces indenting the python code.  (odd, I know)